resources/views/components/yes-no-maybe-buttons.blade.php

```diff
@@ -2,17 +2,71 @@
     @csrf
     <input name="question" type="hidden" value="{{ $quId }}">
     <input name="answer" type="hidden" value="yes">
-    <button accesskey="y" class="bg-green-500 hover:bg-green-700 text-white font-bold py-2 px-4 rounded">Yes</button>
+    <button accesskey="y" class="bg-green-500 hover:bg-green-700 text-white font-bold py-2 px-4 rounded" title="Press 1 to execute">Yes [1]</button>
 </form>
 <form id="no-form" action="{{ route('answers') }}" method="POST">
     @csrf
     <input name="question" type="hidden" value="{{ $quId }}">
     <input name="answer" type="hidden" value="no">
-    <button accesskey="n" class="bg-red-500 hover:bg-red-700 text-white font-bold py-2 px-4 rounded">No</button>
+    <button accesskey="n" class="bg-red-500 hover:bg-red-700 text-white font-bold py-2 px-4 rounded" title="Press 2 to execute">No [2]</button>
 </form>
 <form class="pl-2" id="skip-form" action="{{ route('answers') }}" method="POST">
     @csrf
     <input name="question" type="hidden" value="{{ $quId }}">
     <input name="answer" type="hidden" value="skip">
-    <button accesskey="s" class="bg-blue-500 hover:bg-blue-700 text-white font-bold py-2 px-4 rounded">Skip</button>
+    <button accesskey="s" class="bg-blue-500 hover:bg-blue-700 text-white font-bold py-2 px-4 rounded" title="Press 3 to execute">Skip [3]</button>
 </form>
+
+<script>
+/**
+ * Enhance form with keys	
+ * 
+ * Note this will fail in IE, but will not brake other code.
```

Comment:
Small typo:

```suggestion
 * Note this will fail in IE, but will not break other code.
```

Also, did you perhaps mean something like "but will work in other browsers"? Otherwise it's not clear what "other code" one would expect this to break, nor how that relates to the browser being IE or not.

Reply:
That was before I notice you don't have any other JS code :-)... But I did mean other JS code on the same page. The script tag will brake in IE (it won't understand class syntax), so no code in this script tag will be executed. But if you would add any IE compatible code in a separate script tag, then it will still work in IE.